src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -0,0 +1,570 @@
+        )
+        self.ContainerServiceLinuxProfile = self.cmd.get_models(
+            "ContainerServiceLinuxProfile",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
+        )
+        self.ManagedClusterServicePrincipalProfile = self.cmd.get_models(
+            "ManagedClusterServicePrincipalProfile",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
+        )
+        self.ContainerServiceSshConfiguration = self.cmd.get_models(
+            "ContainerServiceSshConfiguration",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
+        )
+        self.ContainerServiceSshPublicKey = self.cmd.get_models(
+            "ContainerServiceSshPublicKey",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
+        )
+        self.ManagedClusterAADProfile = self.cmd.get_models(
+            "ManagedClusterAADProfile",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
+        )
+        self.ManagedClusterAgentPoolProfile = self.cmd.get_models(
+            "ManagedClusterAgentPoolProfile",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
+        )
+        self.ManagedClusterIdentity = self.cmd.get_models(
+            "ManagedClusterIdentity",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
+        )
+        # pylint: disable=line-too-long
+        self.ComponentsQit0EtSchemasManagedclusterpropertiesPropertiesIdentityprofileAdditionalproperties = self.cmd.get_models(
+            "ComponentsQit0EtSchemasManagedclusterpropertiesPropertiesIdentityprofileAdditionalproperties",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
+        )
+        self.ManagedCluster = self.cmd.get_models(
+            "ManagedCluster",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
+        )
+        # pylint: disable=line-too-long
+        self.Components1Umhcm8SchemasManagedclusteridentityPropertiesUserassignedidentitiesAdditionalproperties = self.cmd.get_models(
+            "Components1Umhcm8SchemasManagedclusteridentityPropertiesUserassignedidentitiesAdditionalproperties",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
+        )
+        self.ExtendedLocation = self.cmd.get_models(
+            "ExtendedLocation",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
+        )
+        self.ExtendedLocationTypes = self.cmd.get_models(
+            "ExtendedLocationTypes",
+            resource_type=self.resource_type,
+            operation_group="managed_clusters",
+        )
+
+
+# pylint: disable=too-few-public-methods
+class AKSCreateParameters:
```

Comment:
I think this is useless?

Reply:
Yeah, removed in commit [58a9a66](https://github.com/Azure/azure-cli/pull/19259/commits/58a9a662d78b0f79364eb38a4f8dd0934366abc1).